diff --git a/lib/browser/driftClient.js b/lib/browser/driftClient.js
index 41a66da1474c34ab67a9deb220993c9df3fb63eb..924b6544e89e04601289615d3ce73b2a56fe025d 100644
--- a/lib/browser/driftClient.js
+++ b/lib/browser/driftClient.js
@@ -70,7 +70,7 @@ const grpcDriftClientAccountSubscriber_1 = require("./accounts/grpcDriftClientAc
 const tweetnacl_1 = __importDefault(require("tweetnacl"));
 const oracleId_1 = require("./oracles/oracleId");
 // BN is already imported globally in this file via other imports
-const sha256_1 = require("@noble/hashes/sha256");
+const sha256_1 = require("@noble/hashes/sha2.js");
 const utils_3 = require("./oracles/utils");
 const orders_1 = require("./math/orders");
 const builder_1 = require("./math/builder");
diff --git a/lib/browser/swift/swiftOrderSubscriber.js b/lib/browser/swift/swiftOrderSubscriber.js
index 4fb011d4758a8f19e611addf38dd0fee44d48fbb..c45c1c7f7ed62363ea95a3320316d1bce4272def 100644
--- a/lib/browser/swift/swiftOrderSubscriber.js
+++ b/lib/browser/swift/swiftOrderSubscriber.js
@@ -11,7 +11,7 @@ const web3_js_1 = require("@solana/web3.js");
 const tweetnacl_1 = __importDefault(require("tweetnacl"));
 const tweetnacl_util_1 = require("tweetnacl-util");
 const ws_1 = __importDefault(require("ws"));
-const sha256_1 = require("@noble/hashes/sha256");
+const sha256_1 = require("@noble/hashes/sha2.js");
 class SwiftOrderSubscriber {
     constructor(config) {
         this.config = config;
diff --git a/lib/node/driftClient.js b/lib/node/driftClient.js
index 41a66da1474c34ab67a9deb220993c9df3fb63eb..924b6544e89e04601289615d3ce73b2a56fe025d 100644
--- a/lib/node/driftClient.js
+++ b/lib/node/driftClient.js
@@ -70,7 +70,7 @@ const grpcDriftClientAccountSubscriber_1 = require("./accounts/grpcDriftClientAc
 const tweetnacl_1 = __importDefault(require("tweetnacl"));
 const oracleId_1 = require("./oracles/oracleId");
 // BN is already imported globally in this file via other imports
-const sha256_1 = require("@noble/hashes/sha256");
+const sha256_1 = require("@noble/hashes/sha2.js");
 const utils_3 = require("./oracles/utils");
 const orders_1 = require("./math/orders");
 const builder_1 = require("./math/builder");
diff --git a/lib/node/swift/swiftOrderSubscriber.js b/lib/node/swift/swiftOrderSubscriber.js
index 4fb011d4758a8f19e611addf38dd0fee44d48fbb..c45c1c7f7ed62363ea95a3320316d1bce4272def 100644
--- a/lib/node/swift/swiftOrderSubscriber.js
+++ b/lib/node/swift/swiftOrderSubscriber.js
@@ -11,7 +11,7 @@ const web3_js_1 = require("@solana/web3.js");
 const tweetnacl_1 = __importDefault(require("tweetnacl"));
 const tweetnacl_util_1 = require("tweetnacl-util");
 const ws_1 = __importDefault(require("ws"));
-const sha256_1 = require("@noble/hashes/sha256");
+const sha256_1 = require("@noble/hashes/sha2.js");
 class SwiftOrderSubscriber {
     constructor(config) {
         this.config = config;
