diff --git a/node_modules/@orca-so/common-sdk/.bun-tag-d1b7d85cd245eea2 b/.bun-tag-d1b7d85cd245eea2
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/web3/token-util.js b/dist/web3/token-util.js
index a1915973084b9f5c69ddad9922b8b5c8c63b46bf..e634274d137a19cb9028afa139bb36d3228b730e 100644
--- a/dist/web3/token-util.js
+++ b/dist/web3/token-util.js
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.TokenUtil = void 0;
 const spl_token_1 = require("@solana/spl-token");
 const web3_js_1 = require("@solana/web3.js");
-const sha256_1 = require("@noble/hashes/sha256");
+const sha256_1 = require("@noble/hashes/sha2");
 const tiny_invariant_1 = __importDefault(require("tiny-invariant"));
 const math_1 = require("../math");
 const web3_1 = require("../web3");
